otx/algorithms/action/adapters/mmaction/utils/export_utils.py

```diff
@@ -0,0 +1,173 @@
+#Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+from subprocess import DEVNULL, CalledProcessError, run  # nosec
+
+try:
+    import onnx
+    import onnxruntime as rt
+    rt.set_default_logger_severity(3)
+except ImportError as e:
+    raise ImportError(f'Please install onnx and onnxruntime first. {e}')
+
+try:
+    from mmcv.onnx.symbolic import register_extra_symbolics
+except ModuleNotFoundError:
+    raise NotImplementedError('please update mmcv to version>=1.0.4')
+
+
+def _convert_batchnorm(module):
```

Comment:
Can you write why we should change batchnorm in this doc string?
And I want to know changing batchnorm affects model's accuracy